Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/CATracker.h

```diff
@@ -10,16 +10,18 @@
 #define ALIITSUCATRACKER_H
 
 #include <vector>
+#include <array>
 
 #include "ITSReconstruction/CAaux.h"
 #include "ITSReconstruction/CATrackingStation.h"
 #include "DetectorsBase/Track.h"
 
-using std::vector;
-
 namespace AliceO2 {
   namespace ITS {
     namespace CA {
+      using std::vector;
```

Comment:
I changed the code to comply with the Coding Guidelines. Honestly, while I see your point Giulio, I also find quite annoying to specify Constants:: before every constant I use in the header files. "Polluting" a very narrow namespace that should not be extended (like the Track namespace, or the CA one) is not that bad in this case. Agreed that for std::array or std::vector this can be avoided.